src/RobustAdaptiveMetropolis.jl

```diff
@@ -0,0 +1,205 @@
+module RobustAdaptiveMetropolis
```

Comment:
I'm defining fairly "general" methods here, e.g. `adapt` and `step_inner`, and so I didn't want to add these to the entire namespace.

Reply:
Hmm I'm not fully convinced as there are not many functions besides the AbstractMCMC definitions and you could easily rename them to `ram_...` (or something else) if necessary. There also doesn't seem any intention currently to split out the module into a separate package. Sometimes a disadvantage with submodules is that it can lead to annoyingly long and complex type names (in e.g. stdout). But I think it's fine either way 🙂

Reply:
Did what you suggested 👍

Reply:
I'm going to leave what in the legal system people would could a "dissenting opinion": I preferred it as a separate module. I think the Julia community should come around to using waaay more submodules to keep namespace pollution down and code more readable. Happy to merge regardless, not a big deal.